backend/src/main/java/com/crm/exception/GlobalExceptionHandler.java

```diff
@@ -36,6 +36,12 @@ public ErrorResponse handleCarNotFoundException(final CarNotFoundException e, fi
         return new ErrorResponse(getSource(request), List.of(e.getMessage()));
     }
 
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
I think there should be HttpStatus.NOT_FOUND

Reply:
I used a wrong method name - it should be 'handleInvalidRegistrationNumberException'. I think it could be also code 400 because 404 is mostly used for searching resources by ID, and registrationNumber is a property of Car, but I will change it to 404 for clarity.